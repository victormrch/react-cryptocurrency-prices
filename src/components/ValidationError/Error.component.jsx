import { TextError } from './Error.style';

export const Error = ({ children }) => {
	return <TextError>{children}</TextError>;
};
