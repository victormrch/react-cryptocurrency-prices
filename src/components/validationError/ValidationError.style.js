import styled from '@emotion/styled';

export const TextError = styled.div`
	background-color: #b7322c;
	color: #fff;
	padding: 15px;
	font-size: 15px;
	text-transform: uppercase;
	font-family: 'Lato', sans-serif;
	font-weight: 700;
	text-align: center;
`;
