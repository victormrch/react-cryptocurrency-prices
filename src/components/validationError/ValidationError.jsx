import { TextError } from './ValidationError.style';

export const ValidationError = ({ children }) => {
	return <TextError>{children}</TextError>;
};
