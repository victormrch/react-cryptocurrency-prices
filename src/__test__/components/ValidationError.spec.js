import { render } from '@testing-library/react';
import { ValidationError } from '../../components/validationError/ValidationError';
import '@testing-library/jest-dom/extend-expect';

describe('ValidateError component specs', () => {
	it('should render correctly using snapshot testing', () => {
		const { asFragment } = render(<ValidationError />);
		expect(asFragment()).toMatchSnapshot();
	});
});
