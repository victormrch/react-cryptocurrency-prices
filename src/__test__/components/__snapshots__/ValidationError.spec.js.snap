// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValidateError component specs should render correctly using snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="css-rfyw22"
  />
</DocumentFragment>
`;
